tracer/test/test-applications/integrations/Samples.ServiceStack.Redis/Samples.ServiceStack.Redis.csproj

```diff
@@ -1,7 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <ApiVersion Condition="'$(ApiVersion)' == ''">5.2.0</ApiVersion>
+    <TargetFrameworks>net461;netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
```

Comment:
If we want to still build/test older package versions like 6.1.0 or 5.2.0 on netcoreapp3.0, I recommend removing both changes in this file so we still build the application for all of the frameworks, and we can test locally in VS by setting `ApiVersion` to a version that's compatible with all of the TFM's we test against.

The changes in `tracer/build/PackageVersionsGeneratorDefinitions.json` alone should be enough to ensure that CI won't build an incompatible package version/TFM

Reply:
Ah yeah good point on that this removes the ability to test older versions
The issue that I was having with this specifically is that when I bumped the version to `6.2.0` it would attempt to build NET Core 2.1/3.0 here (which would fail). 

I'll see if making Framework conditionals for the `<ItemGroup>` to have multiple `PackageReference` entries for ServiceStack does the trick.

Reply:
I've swapped it to use two conditional `ItemGroup` one for .NET Core 2.1/3.0 and one for anything other than those.

Let me know if that looks good otherwise I can just revert it back to the original.

I was thinking that this way wouldn't work at first because there's this `<GenerateTargetFrameworkAttribute>false</GenerateTargetFrameworkAttribute>` in the sample csproj and I thought that'd break the `$(TargetFramework)` value, but apparently not.

Reply:
Oh right, the `<GenerateTargetFrameworkAttribute>false</GenerateTargetFrameworkAttribute>` property controls whether the C# compilation should generate a `[TargetFrameworkAttribute]` that is applied to the resulting assembly. All the `<GenerateXXXAttribute>` flags operate just like this, so it doesn't affect the MSBuild properties but assembly attributes on the compiled application 👍🏼